lib/process.ts

```diff
@@ -1,15 +1,20 @@
-// @ts-nocheck
-const readline = require('readline');
-const {
-  logger,
-  setLogLevel,
-  LOG_LEVEL,
-} = require('@hubspot/local-dev-lib/logger');
-const { i18n } = require('./lang');
+import readline from 'readline';
+import { logger, setLogLevel, LOG_LEVEL } from '@hubspot/local-dev-lib/logger';
+import { i18n } from './lang';
 
 const i18nKey = 'lib.process';
 
-const handleExit = callback => {
+interface KeyPress {
+  ctrl?: boolean;
+  shift?: boolean;
+  alt?: boolean;
+  meta?: boolean;
+  name?: string;
+}
+
+export function handleExit(
+  callback: (onTerminate?: { isSIGHUP: boolean }) => void
```

Comment:
I think the `onTerminate` field should be required here instead of optional. The callback will always be passed `isSIGHUP` within this function, regardless of whether the calling code actually leverages it or not. I noticed this when trying to port a file that uses this `handleExit` util to TS and got an error.